webapp/babel.config.js

```diff
@@ -1,6 +1,3 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
-// See LICENSE.txt for license information.
-
```

Comment:
Why remove these comments?

Reply:
Not needed. Were added while syncing with starter template.